documentation/cloud-manager-guide/admin/integrations/manage-integrations.md

```diff
@@ -5,4 +5,71 @@ description: Manage integrations
 ---
 # Manage Integrations
 
-**This page is a WIP - to be included with beta release**
+External communication channels can be integrated with Environment Operations Center to receive alerts or notifications about changes of state. From the *Integrations* tab in the Admin section, Administrators can add, edit, or delete integrations. This guide outlines the required steps to manage communication channel integrations.
+
+## Getting started
+
+The *Integrations* tab provides an overview of existing integrations and lists the label, integration type, and channel for each integration. 
+
+![image description](images/home.png)
+
+## Configure a new integration
+
+To add a new integration, select **New Integration** which opens the *Integration Setup* dialog. To complete the integration setup, you must provide an integration type and configuration details.
+
+![image description](images/new-int.png)
+
+### Integration type
+
+Select an **Integration Type** from the available channels listed and select **Next** to continue.
+
+![image description](images/type.png)
+
+### Configuration Details
+
+The required configuration details will differ depending on the type of configuration selected. For Slack specifically, the integration name, slack workspace, channel, and your email are required.
+
+Complete the required configuration fields and select **Create** to finish setting up the new integration.
+
+![image description](images/config.png)
```

Comment:
When we do the final screens, suggest editing out mentions of Slack so that the screen shot doesn't need to be updated if the slack integration requirements change. Making it generic is good enough as we say each integration has unique requirements.